subprojects/tooling-api/src/main/java/org/gradle/tooling/model/idea/IdeaContentRoot.java

```diff
@@ -58,6 +58,29 @@
     @Incubating
     DomainObjectSet<? extends IdeaSourceDirectory> getGeneratedTestDirectories();
 
+    /**
+     * The set of resource directories.
+     */
+    DomainObjectSet<? extends IdeaSourceDirectory> getResourceDirectories();
```

Comment:
The [IDEA documentation](https://www.jetbrains.com/help/idea/working-with-modules.html) explicitly states that The resources and test resources directory is only available for Java projects. I think we should explicitly specify this use case in the javadoc.

Reply:
added a note to the javadoc